### PR TITLE
Parklet

### PR DESCRIPTION
**Pull request by mharegot**
support for parklet model. Intending to get parklet asset fully functional within streetmix. @srothst1 will review

### CODE REVIEW

File: src/aframe-streetmix-parsers.js [hunk 1]
```diff
@@ -631,6 +631,8 @@ function processSegments (segments, showStriping, length) {
     } else if (segments[i].type === 'outdoor-dining') {
       groundMixinId = (variantList[1] === 'road') ? 'drive-lane' : 'sidewalk';
       segmentParentEl.append(createOutdoorDining(variantList, positionX));
+    } else if (segments[i].type === 'parklet'){
+      console.log("parklet");
```
Comment (srothst1) [inline src/aframe-streetmix-parsers.js:635]: Nice work @mharegot :+1: As we discussed earlier today, we will pick up where we left off and implement a function that adds a parklet to the scene.

**Comment by srothst1** [comment]
image

**Comment by srothst1** [comment]
@mharegot thank you again for your contributions :grinning: :rocket: 

This pull request addresses #11. @kfarr it looks like there are some merge conflicts that must be resolved. The conflicts are in the following files:
- `dist/aframe-street-component.js`
- `package-lock.json`
I'm assuming that we can just delete this pull requests commits to those two files. Let me know what you think.

**Comment by kfarr** [comment]
Thanks @mharegot and @srothst1! Yes I agree we can just delete these two files and it should merge OK. I will see if I can do that next.

**Pull request merged**